utils/customize_node.py

```diff
@@ -67,10 +67,17 @@ def extract_bootfiles(args, phase_msg):
             set_trace()
         raise RuntimeError('Multiple (%d) %s kernels exist' % (
             len(vmlinuz), phase_msg))
+
     if not hasattr(args, 'vmlinuz_golden'):         # singleton
         args.vmlinuz_golden = ''
+
+    if phase_msg == 'golden':
+        update_status(args, 'Kernel found. Keeping as is without moving...')
```

Comment:
LIst the precise file name that's being kept.  It helps in debugging and post mortem analysis.